### PR TITLE
Fix switch

### PR DESCRIPTION
**Pull request by xhuberty**

### CODE REVIEW

File: src/SwiftTwigMailTemplate.php [hunk 1]
```diff
@@ -131,28 +131,50 @@ public function renderMail(array $data = []) :\Swift_Message
...
+                case $this->priority && $i < 7 :
+                    $i = 7;
+                    $mail->setPriority($this->priority);
+                    break;
+                case $this->readReceiptTo && $i < 8 :
+                    $i = 8;
+                    $mail->setReadReceiptTo($this->readReceiptTo);
+                    break;
+                case $this->returnPath && $i < 9 :
+                    $i = 9;
```
Comment (Ngob) [inline src/SwiftTwigMailTemplate.php]: $i = 10 or break 2; ?

**Comment by moufmouf** [comment]
Seriously?

Instead of 10 simple to understand `if` statements?

If this is what Scrutinizer is leading you to write, I suggest stopping using Scrutinizer right now :sob: 

Or start participating to the IOCCC :)

**Comment by moufmouf** [comment]
Haaaa, nice :smile:

**Comment by moufmouf** [comment]
(and fuck Scrutinizer, this is the right way to do things!)

**Pull request merged**